content/en/_index.md

```diff
@@ -96,6 +106,12 @@ the sample work:
 * See https://cloud.google.com/compute/docs/quickstart-client-libraries before running the code snippet.
 */
 {{< /tab >}}
+{{< tab header="Ruby" >}}
+  
+  # Moves a persistent disk from one zone to another.
+  #
+  # See https://cloud.google.com/compute/docs/quickstart-client-libraries before running the code snippet.
+{{< /tab >}}
 {{< /tabpane >}}
 
 ### Main runner
```

Comment:
I think we are missing a sample from this section. I will send a PR to update the terminology

Reply:
updated this.. seems it got mixed up when resolving conflicts